.github/workflows/perl.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: Perl authortests
+
+on: [push, pull_request]
+
+jobs:
+  authortests:
+    runs-on: ubuntu-latest
+    name: Perl
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/os-autoinst_dev
+    steps:
+      - uses: actions/checkout@v4
+      - run: git config --global --add safe.directory '*'
+      - run: prove xt
```

Comment:
I would prefer not to pile up more github action specific implementations because then developers won't know how to run all tests and checks locally, see https://github.com/os-autoinst/os-autoinst-common/pull/46

Reply:
Well, maybe we shouldn't work on this in parallel with 3 persons...